code/controllers/configuration.dm

```diff
@@ -258,6 +258,8 @@
 	var/arrivals_shuttle_dock_window = 55	//Time from when a player late joins on the arrivals shuttle to when the shuttle docks on the station
 	var/arrivals_shuttle_require_safe_latejoin = FALSE	//Require the arrivals shuttle to be operational in order for latejoiners to join
 
+	var/mice_roundstart = 10 // how many wire chewing rodents spawn at roundstart.

```

Comment:
No, it has to be 10, this adds a new config option so it's suppose to preserve old behavior. Normally its 10, changing it to 20 would have to be in another pr